src/sentry/static/sentry/app/views/workBatchList/index.jsx

```diff
@@ -1,48 +1,81 @@
-import PropTypes from 'prop-types';
 import React from 'react';
-import withEnvironmentInQueryString from 'app/utils/withEnvironmentInQueryString'; // REMOVE ME
-import WorkBatches from 'app/views/workBatchList/workBatches';
+import withOrganization from 'app/utils/withOrganization';
 import {connect} from 'react-redux';
-import {tagsGet} from 'app/redux/actions/tag';
-// TODO: uncomment these when fixing CLIMS-203
-// import {Client} from 'app/api';
-// import {fetchOrgMembers} from 'app/actionCreators/members';
+import {workBatchActions} from 'app/redux/actions/workBatch';
+import ClimsTypes from 'app/climsTypes';
+import ListViewContainer from 'app/components/listViewContainer';
+import moment from 'moment';
+import Link from 'app/components/link';
 
-class WorkBatchList extends React.Component {
+class WorkBatchListContainer extends React.Component {
```

Comment:
The name of the file is different. Should it match?

Reply:
I agree. I think it would be better if we synch it like you suggest, but currently this convention is in place in a lot of the Sentry code, i.e. the "connecting" component, the one that sets up things as required by react, is in a file called "index.jsx". I would not be against refactoring that all over, as I'm tired of figuring out where I am based on the path of the file rather than the name of the file!